### DISCUSSION TITLE
using <netinet/in.h>

### DISCUSSION THREAD

**Discussion by WLS04** [ANLAB-KAIST/KENSv3#250] [Project Q&A 3. Internet Layer (IP/Routing)]
이전 과제들과 다르게 `<netinet/in.h>`가 include되어 있지 않은 것 같습니다.
htonl, ntohl 등을 사용하려면 필요할 것 같은데 `<netinet/in.h>` 를 사용해도 될까요? 
아니면 이번 과제에서는 사용하면 안되는 것인가요?

**Comment by Pusnow** [comment]
You can use it and other POSIX/Linux headers
